indexer/packages/postgres/__tests__/stores/wallet-table.test.ts

```diff
@@ -1,6 +1,6 @@
 import { WalletFromDatabase } from '../../src/types';
 import { clearData, migrate, teardown } from '../../src/helpers/db-helpers';
-import { defaultAddress, defaultWallet } from '../helpers/constants';
+import { defaultWallet2 } from '../helpers/constants';
```

Comment:
is this necessary?

Reply:
Since `defaultWallet` is created by default now, and half of the functions in `wallet-table` are testing behavior before and after the wallet is created, it was easiest to just use `defaultWallet2` instead of trying to customize the logic